webui/react/src/components/kit/Card.tsx

```diff
@@ -0,0 +1,90 @@
+import { Dropdown, MenuProps, Space } from 'antd';
+import React from 'react';
+
+import { ConditionalWrapper } from 'components/ConditionalWrapper';
+import Link from 'components/Link';
+import Icon from 'shared/components/Icon';
+
+import Button from './Button';
+import css from './Card.module.scss';
+
+interface CardProps {
+  actionMenu?: MenuProps;
+  children?: React.ReactNode;
+  clickable?: boolean;
+  disabled?: boolean;
+  height?: number;
+  href?: string;
+  onClick?: () => void;
+  width?: number;
+}
+
+const stopPropagation = (e: React.MouseEvent): void => e.stopPropagation();
+
+const Card: Card = ({
+  actionMenu,
+  children,
+  clickable,
+  disabled = false,
+  onClick,
+  height = 184,
```

Comment:
Do we really want to be able to specify arbitrary dimensions? Also how does that work with responsive design?